firmware/Core/Inc/mpi/mpi_types.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * mpi_types.h
+ *
+ * Created on: Jun 14, 2024
+ * Author: Vaibhav Kapoor
+ */
+
+#ifndef __INCLUDE_GUARD__MPI_TYPES_H
+#define __INCLUDE_GUARD__MPI_TYPES_H
+#include <stdint.h>
+
+/**
+ * @enum Mpi_Mode
+ * @brief Enumerates the different modes under which the satellite can operate the MPI
+ * 
+ */
+enum Mpi_Mode {
+    COMMAND_MODE,                           // MPI is in command mode
```

Comment:
Please prefix each enum option with something like `MPI_RX_MODE_` (e.g., `MPI_RX_MODE_SCIENCE_DATA`, etc.)

Reply:
Comment not implement. Please implement comment.